src/nvim/viml/parser/expressions.c

```diff
@@ -452,6 +464,8 @@ LexExprToken viml_pexpr_next_token(ParserState *const pstate, const bool peek)
 //
 // Used highlighting groups and assumed linkage:
 //
+// NVimInternalError -> highlight as fg:red/bg:red
```

Comment:
Is this correct, fg and bg having the same color?

Reply:
As an internal error it should be something which will make user report it. Given that I do not want to spawn disturbing errors “invisible”, outstanding character is a good alternative.

Reply:
Note that this is also the only group which I did not bother to (plan to) link to some preexisting group.